src/services/BrowserActionService.ts

```diff
@@ -175,7 +175,7 @@ export const checkIfProtected = async (
           break;
       }
     } else {
-      if (cookieLength && cookieLength === 0) {
+      if (cookieLength !== undefined && cookieLength === 0) {
```

Comment:
I don't understand. Isn't doing a truthy check or cookieLength good enough? As it is checking for both undefined and null?

Reply:
I had left a note in the PR comment but I'll re-iterate here:

when cookieLength is undefined OR defined as number 0, both return false in conditionals.

Reply:
oh wait...did I just correct myself there...